compiler/modules/front.syntax/parseStmt.bal

```diff
@@ -160,39 +159,43 @@ function finishIdentifierStmt(Tokenizer tok, string identifier, Position pos, Po
             endPos = check tok.expectEnd(";");
             return { startPos, endPos, expr };
         }
-        return parseError(tok, "member access expr not allowed as a statement"); 
+        return parseError(tok, "member access expr not allowed as a statement");
     }
     else if cur == ":" {
         check tok.advance();
-        return finishOptQualIdentifierStmt(tok, identifier, check tok.expectIdentifier(), pos, startPos);
+        Position namePos = tok.currentStartPos();
+        string name = check tok.expectIdentifier();
```

Comment:
I would inline this, as before.

The naming of variables/parameters needs work here and below. Using both `name` and `identifier` is confusing.  If you want to distinguish the thing after the `:`, call it the _local_ name (the name local to the module).

Reply:
fixed